staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/converter.go

```diff
@@ -50,28 +56,30 @@ func NewCRConverterFactory(serviceResolver webhook.ServiceResolver, authResolver
 	return converterFactory, nil
 }
 
-// NewConverter returns a new CR converter based on the conversion settings in crd object.
-func (m *CRConverterFactory) NewConverter(crd *apiextensionsv1.CustomResourceDefinition) (safe, unsafe runtime.ObjectConvertor, err error) {
-	validVersions := map[schema.GroupVersion]bool{}
-	for _, version := range crd.Spec.Versions {
-		validVersions[schema.GroupVersion{Group: crd.Spec.Group, Version: version.Name}] = true
-	}
-
-	var converter CRConverter
+// NewConverter creates a new CRConverter based on the crd's conversion strategy. Supported strategies are none and
+// webhook.
+func (f *CRConverterFactory) NewConverter(crd *apiextensionsv1.CustomResourceDefinition) (CRConverter, error) {
```

Comment:
are callers going to get bitten by this returning a convertor that *has* to be wrapped by a separate call to NewDelegatingConverter to work properly?

Reply:
Maaaaybe but the way this code is currently factored, this was the best "least" amount of work I could come up with.